src/main/scala/org/broadinstitute/dsde/agora/server/acls/GcsRole.scala

```diff
@@ -0,0 +1,30 @@
+package org.broadinstitute.dsde.agora.server.acls
+
+import org.broadinstitute.dsde.agora.server.acls.GcsRole._
+
+/**
+ * Created by dshiga on 6/16/15.
+ */
+
```

Comment:
We don't need author tags. All of the information we need about who had done what is in github and at a much more granular level. This is probably code template in your Intellij for new classes that you can just set to blank.